near-sdk/src/utils/mod.rs

```diff
@@ -63,6 +91,7 @@ impl PendingContractTx {
 }
 
 /// Boilerplate for setting up allocator used in Wasm binary.
+/// This initializes the allocator with [`WeeAlloc`](crate::wee_alloc::WeeAlloc).
```

Comment:
Let's maybe rephrase this to something like `Sets up the [global allocator](link to std) to WeeAlloc`. The current phrasing lacks the explanation/like for what an allocator is.